Tests/BasicsTests/AsyncProcessTests.swift

```diff
@@ -561,12 +532,25 @@ extension AsyncProcess {
         outputRedirection: OutputRedirection = .collect
     ) {
         self.init(
-            arguments: [Self.script(scriptName)] + arguments,
+            arguments: getAsyncProcessArgs(executable: AsyncProcess.script(scriptName)) + arguments,
             environment: .current,
             outputRedirection: outputRedirection
         )
     }
 
+    @discardableResult
+    fileprivate static func checkNonZeroExit(
```

Comment:
ditto, what's the value of this function if only thing it does is renaming `arguments` to `args`?

Reply:
The underlying issue is Swift does not allow to easily convert an array to a variadic.